mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a,b,c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age){
+  return `Hello, my name is ${name} and I am ${age} years old`;
+}
 
 function getTotal(a, b) {
-  total = a ++ b;
-
-  return "The total is total";
+  let total = a + b;
+  return `The total is ${total}`;
 }
```

Comment:
That's right. Ass you see, you cannot use `++` to add to String values together, `++` can only be used while increasing the value of a variable by 1. Other than that, as you did on this question, you always need to use either the let or the const keyword while defining a variable, otherwise JavaScript won't understand that it is a variable, and won't save it on its memory.